### PR TITLE
[DOJOT-150] add: Grafana dashboards with InfluxDB telemetry data.

### PR DESCRIPTION
**Pull request by ademirr**
…ata.

* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

[DOJOT-150] is an epic with many stories. The stories referring to the grafana dashboards with InfluxDB telemetry data are:

[DOJOT-156]
[DOJOT-157]
[DOJOT-158]
[DOJOT-161]
[DOJOT-163]
[DOJOT-164]
[DOJOT-165]
[DOJOT-166]
[DOJOT-167]
[DOJOT-168]
[DOJOT-169]
[DOJOT-170]

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

https://dojot.atlassian.net/browse/DOJOT-156
https://dojot.atlassian.net/browse/DOJOT-157
https://dojot.atlassian.net/browse/DOJOT-158
https://dojot.atlassian.net/browse/DOJOT-161
https://dojot.atlassian.net/browse/DOJOT-163
https://dojot.atlassian.net/browse/DOJOT-164
https://dojot.atlassian.net/browse/DOJOT-165
https://dojot.atlassian.net/browse/DOJOT-166
https://dojot.atlassian.net/browse/DOJOT-167
https://dojot.atlassian.net/browse/DOJOT-168
https://dojot.atlassian.net/browse/DOJOT-169
https://dojot.atlassian.net/browse/DOJOT-170

* **Other information**:

[DOJOT-150]: https://dojot.atlassian.net/browse/DOJOT-150?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOJOT-156]: https://dojot.atlassian.net/browse/DOJOT-156?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOJOT-157]: https://dojot.atlassian.net/browse/DOJOT-157?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOJOT-158]: https://dojot.atlassian.net/browse/DOJOT-158?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOJOT-161]: https://dojot.atlassian.net/browse/DOJOT-161?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOJOT-163]: https://dojot.atlassian.net/browse/DOJOT-163?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOJOT-164]: https://dojot.atlassian.net/browse/DOJOT-164?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOJOT-165]: https://dojot.atlassian.net/browse/DOJOT-165?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOJOT-166]: https://dojot.atlassian.net/browse/DOJOT-166?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOJOT-167]: https://dojot.atlassian.net/browse/DOJOT-167?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOJOT-168]: https://dojot.atlassian.net/browse/DOJOT-168?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOJOT-169]: https://dojot.atlassian.net/browse/DOJOT-169?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOJOT-170]: https://dojot.atlassian.net/browse/DOJOT-170?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: data-visualization/grafana-templates/datasource.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - name: PostgreSQL
+    type: PostgreSQL
+    url: localhost:5432
+    database: dojot_devm
+    user: postgres
+    secureJsonData:
+      password: "postgres"
+    jsonData:
+      sslmode: "disable" # disable/require/verify-ca/verify-full
+      maxOpenConns: 0         # Grafana v5.4+
```
Comment (gabrielifg) [inline data-visualization/grafana-templates/datasource.yml:13]: Uma sugestão, para deixar o código mais limpo, você pode tirar esses comentários "repetidos"

**Review by gabrielifg — APPROVED** [review]
Excelente trabalho!!!

**Review by rodrigoaasm — APPROVED** [review]
Ok

File: data-visualization/grafana-templates/datasource.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+apiVersion: 1
+
+datasources:
+  - name: PostgreSQL
+    type: PostgreSQL
```
Comment (gabrielifg) [inline data-visualization/grafana-templates/datasource.yml:5]: O valor do type, deve estar dessa forma "postgres"

File: data-visualization/grafana-templates/datasource.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    jsonData:
+      sslmode: "disable" # disable/require/verify-ca/verify-full
+      maxOpenConns: 0         # Grafana v5.4+
+      maxIdleConns: 2         # Grafana v5.4+
+      connMaxLifetime: 14400  # Grafana v5.4+
+      postgresVersion: 903 # 903=9.3, 904=9.4, 905=9.5, 906=9.6, 1000=10
+      timescaledb: false
+
+  - name: InfluxDB
+    type: InfluxDB
```
Comment (gabrielifg) [inline data-visualization/grafana-templates/datasource.yml:20]: O valor do type, deve estar dessa forma "influxdb"

File: data-visualization/grafana-templates/datasource.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    user: postgres
+    secureJsonData:
+      password: "postgres"
+    jsonData:
+      sslmode: "disable" # disable/require/verify-ca/verify-full
+      maxOpenConns: 0         # Grafana v5.4+
+      maxIdleConns: 2         # Grafana v5.4+
+      connMaxLifetime: 14400  # Grafana v5.4+
+      postgresVersion: 903 # 903=9.3, 904=9.4, 905=9.5, 906=9.6, 1000=10
+      timescaledb: false
```
Comment (gabrielifg) [inline data-visualization/grafana-templates/datasource.yml:17]: Seria interessante adicionar ao final de ambos os datasource essas intruções para permitir a auteração e a escrita.

      readOnly: false
      editable: true

**Review by gabrielifg — CHANGES_REQUESTED** [review]

**Review by rodrigoaasm — APPROVED** [review]
Approved!

**Review by gabrielifg — APPROVED** [review]

**Pull request merged**